docs/magento1/installation.mdx

```diff
@@ -225,7 +225,7 @@ For more security add a random key on `frontcommerce_secret_key` in your
 
 ### Clean Magento cache
 
-## Ensure it works Once this is done, all the checks should be green in your
+## Ensure it works Once this is done, all the checks should be green in your
```

Comment:
that was a non breaking space resulting into:

![image](https://user-images.githubusercontent.com/305563/213246492-7f81ee7f-841b-428e-8ad3-e90dde4fecb6.png)